maquetado/scss/tech.scss

```diff
@@ -0,0 +1,68 @@
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+%card {
+	height: 20em;
+	border: 1px solid $green;
+	border-radius: 5px;
+	margin: 15px 15px;
+	width: 14em;
+}
+
+.container {
+	display: flex;
+	justify-content: center;
+	width: 80%;
+}
+
+.cards {
+	display: flex;
+	flex-wrap: wrap;
+	width: 80%;
+}
+
+.border-ext {
+	margin: 20px 20px;
+	@extend %card;
```

Comment:
This is an strange way to implement this, its not bad tho. I would use a pseudo element instead, it will be harder for you to implement transitions this way